Users/agarwalshiv/datadog/description update

shivansh257 opened the pull request:
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
<!--- Please provide the related command with az {command} if you can, so that we can quickly route to the related person to review. --->


### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).


### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

yonzhan commented:
Thank you for your contribution! We will review the pull request and get back to you soon.

shivansh257 commented on src/datadog/README.md:
```diff
@@ -1,5 +1,5 @@
 
-This package is for the 'datadog' extension.
-i.e. 'az datadog'
+## How to use ##
```
updated

shivansh257 commented:
> ❌Azure CLI Extensions Breaking Change Test
> > ❌datadog
> > 
> > 
> > 
> > rule
> > cmd_name
> > rule_message
> > suggest_message
> > 
> > 
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog monitor create
> > cmd `datadog monitor create` removed parameter `sku_name`
> > please add back parameter `sku_name` for cmd `datadog monitor create`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog monitor create
> > cmd `datadog monitor create` removed parameter `type_`
> > please add back parameter `type_` for cmd `datadog monitor create`
> > 
> > 
> > ❌ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `name`: added property `required=True`
> > please remove property `required=True` for parameter `name` of cmd `datadog monitor set-default-key`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog monitor update
> > cmd `datadog monitor update` removed parameter `sku_name`
> > please add back parameter `sku_name` for cmd `datadog monitor update`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` removed parameter `properties`
> > please add back parameter `properties` for cmd `datadog sso-config create`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` removed parameter `properties`
> > please add back parameter `properties` for cmd `datadog sso-config update`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` removed parameter `filtering_tags`
> > please add back parameter `filtering_tags` for cmd `datadog tag-rule create`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` removed parameter `log_rules_filtering_tags`
> > please add back parameter `log_rules_filtering_tags` for cmd `datadog tag-rule create`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` removed parameter `send_aad_logs`
> > please add back parameter `send_aad_logs` for cmd `datadog tag-rule create`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` removed parameter `send_resource_logs`
> > please add back parameter `send_resource_logs` for cmd `datadog tag-rule create`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` removed parameter `send_subscription_logs`
> > please add back parameter `send_subscription_logs` for cmd `datadog tag-rule create`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` removed parameter `filtering_tags`
> > please add back parameter `filtering_tags` for cmd `datadog tag-rule update`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` removed parameter `log_rules_filtering_tags`
> > please add back parameter `log_rules_filtering_tags` for cmd `datadog tag-rule update`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` removed parameter `send_aad_logs`
> > please add back parameter `send_aad_logs` for cmd `datadog tag-rule update`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` removed parameter `send_resource_logs`
> > please add back parameter `send_resource_logs` for cmd `datadog tag-rule update`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` removed parameter `send_subscription_logs`
> > please add back parameter `send_subscription_logs` for cmd `datadog tag-rule update`
> > 
> > 
> > ❌ [1007 - ParaRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1007.md)
> > datadog terms create
> > cmd `datadog terms create` removed parameter `properties`
> > please add back parameter `properties` for cmd `datadog terms create`
> > 
> > 
> > ❌ [1002 - CmdRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1002.md)
> > datadog terms update
> > cmd `datadog terms update` removed
> > please confirm cmd `datadog terms update` removed
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor create
> > cmd `datadog monitor create` added parameter `identity`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor create
> > cmd `datadog monitor create` added parameter `monitoring_status`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor create
> > cmd `datadog monitor create` added parameter `sku`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `datadog_organization_properties`: added property `aaz_type=AAZObjectArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `datadog_organization_properties`: added property `type=Object`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `location`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `location`: updated property `type` from `custom_type` to `string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `no_wait`: added property `aaz_type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `no_wait`: added property `choices=['0', '1', 'f', 'false', 'n', 'no', 't', 'true', 'y', 'yes']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `no_wait`: added property `nargs=?`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `no_wait`: added property `type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `tags`: added property `aaz_type=AAZDictArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `tags`: added property `type=Dict<String,String>`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `tags`: updated property `nargs` from `*` to `+`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `user_info`: added property `aaz_type=AAZObjectArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor create
> > cmd `datadog monitor create` update parameter `user_info`: added property `type=Object`
> > 
> > 
> > 
> > ⚠️ [1004 - CmdPropRemove](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1004.md)
> > datadog monitor delete
> > cmd `datadog monitor delete` removed property `confirmation`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor delete
> > cmd `datadog monitor delete` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor delete
> > cmd `datadog monitor delete` update parameter `no_wait`: added property `aaz_type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor delete
> > cmd `datadog monitor delete` update parameter `no_wait`: added property `choices=['0', '1', 'f', 'false', 'n', 'no', 't', 'true', 'y', 'yes']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor delete
> > cmd `datadog monitor delete` update parameter `no_wait`: added property `nargs=?`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor delete
> > cmd `datadog monitor delete` update parameter `no_wait`: added property `type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor delete
> > cmd `datadog monitor delete` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor delete
> > cmd `datadog monitor delete` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor delete
> > cmd `datadog monitor delete` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor get-default-key
> > cmd `datadog monitor get-default-key` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor get-default-key
> > cmd `datadog monitor get-default-key` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor get-default-key
> > cmd `datadog monitor get-default-key` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor get-default-key
> > cmd `datadog monitor get-default-key` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list
> > cmd `datadog monitor list` added parameter `pagination_limit`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list
> > cmd `datadog monitor list` added parameter `pagination_token`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list
> > cmd `datadog monitor list` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list
> > cmd `datadog monitor list` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor list
> > cmd `datadog monitor list` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list-api-key
> > cmd `datadog monitor list-api-key` added parameter `pagination_limit`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list-api-key
> > cmd `datadog monitor list-api-key` added parameter `pagination_token`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-api-key
> > cmd `datadog monitor list-api-key` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-api-key
> > cmd `datadog monitor list-api-key` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-api-key
> > cmd `datadog monitor list-api-key` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor list-api-key
> > cmd `datadog monitor list-api-key` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list-host
> > cmd `datadog monitor list-host` added parameter `pagination_limit`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list-host
> > cmd `datadog monitor list-host` added parameter `pagination_token`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-host
> > cmd `datadog monitor list-host` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-host
> > cmd `datadog monitor list-host` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-host
> > cmd `datadog monitor list-host` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor list-host
> > cmd `datadog monitor list-host` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list-linked-resource
> > cmd `datadog monitor list-linked-resource` added parameter `pagination_limit`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list-linked-resource
> > cmd `datadog monitor list-linked-resource` added parameter `pagination_token`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-linked-resource
> > cmd `datadog monitor list-linked-resource` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-linked-resource
> > cmd `datadog monitor list-linked-resource` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-linked-resource
> > cmd `datadog monitor list-linked-resource` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor list-linked-resource
> > cmd `datadog monitor list-linked-resource` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list-monitored-resource
> > cmd `datadog monitor list-monitored-resource` added parameter `pagination_limit`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor list-monitored-resource
> > cmd `datadog monitor list-monitored-resource` added parameter `pagination_token`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-monitored-resource
> > cmd `datadog monitor list-monitored-resource` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-monitored-resource
> > cmd `datadog monitor list-monitored-resource` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor list-monitored-resource
> > cmd `datadog monitor list-monitored-resource` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor list-monitored-resource
> > cmd `datadog monitor list-monitored-resource` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor refresh-set-password-link
> > cmd `datadog monitor refresh-set-password-link` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor refresh-set-password-link
> > cmd `datadog monitor refresh-set-password-link` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor refresh-set-password-link
> > cmd `datadog monitor refresh-set-password-link` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor refresh-set-password-link
> > cmd `datadog monitor refresh-set-password-link` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` added parameter `key_name`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `created_by`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `created`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `key`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `name`: added property `id_part=name`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `name`: updated property `name` from `name` to `monitor_name`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `name`: updated property `options` from `['--name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor set-default-key
> > cmd `datadog monitor set-default-key` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor show
> > cmd `datadog monitor show` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor show
> > cmd `datadog monitor show` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor show
> > cmd `datadog monitor show` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor show
> > cmd `datadog monitor show` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor update
> > cmd `datadog monitor update` added parameter `generic_update_add`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor update
> > cmd `datadog monitor update` added parameter `generic_update_force_string`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor update
> > cmd `datadog monitor update` added parameter `generic_update_remove`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor update
> > cmd `datadog monitor update` added parameter `generic_update_set`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor update
> > cmd `datadog monitor update` added parameter `identity`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor update
> > cmd `datadog monitor update` added parameter `location`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor update
> > cmd `datadog monitor update` added parameter `sku`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog monitor update
> > cmd `datadog monitor update` added parameter `user_info`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `monitoring_status`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `monitoring_status`: added property `choices=['Disabled', 'Enabled']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `no_wait`: added property `aaz_type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `no_wait`: added property `choices=['0', '1', 'f', 'false', 'n', 'no', 't', 'true', 'y', 'yes']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `no_wait`: added property `nargs=?`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `no_wait`: added property `type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `tags`: added property `aaz_type=AAZDictArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `tags`: added property `type=Dict<String,String>`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor update
> > cmd `datadog monitor update` update parameter `tags`: updated property `nargs` from `*` to `+`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor wait
> > cmd `datadog monitor wait` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor wait
> > cmd `datadog monitor wait` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog monitor wait
> > cmd `datadog monitor wait` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog monitor wait
> > cmd `datadog monitor wait` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` added parameter `enterprise_app_id`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` added parameter `single_sign_on_state`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `configuration_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `no_wait`: added property `aaz_type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `no_wait`: added property `choices=['0', '1', 'f', 'false', 'n', 'no', 't', 'true', 'y', 'yes']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `no_wait`: added property `nargs=?`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `no_wait`: added property `type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config create
> > cmd `datadog sso-config create` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog sso-config list
> > cmd `datadog sso-config list` added parameter `pagination_limit`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog sso-config list
> > cmd `datadog sso-config list` added parameter `pagination_token`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config list
> > cmd `datadog sso-config list` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config list
> > cmd `datadog sso-config list` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config list
> > cmd `datadog sso-config list` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config list
> > cmd `datadog sso-config list` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config list
> > cmd `datadog sso-config list` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config show
> > cmd `datadog sso-config show` update parameter `configuration_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config show
> > cmd `datadog sso-config show` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config show
> > cmd `datadog sso-config show` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config show
> > cmd `datadog sso-config show` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config show
> > cmd `datadog sso-config show` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config show
> > cmd `datadog sso-config show` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` added parameter `enterprise_app_id`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` added parameter `single_sign_on_state`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `configuration_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `no_wait`: added property `aaz_type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `no_wait`: added property `choices=['0', '1', 'f', 'false', 'n', 'no', 't', 'true', 'y', 'yes']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `no_wait`: added property `nargs=?`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `no_wait`: added property `type=bool`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `properties_to_add`: added property `aaz_type=AAZGenericUpdateAddArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `properties_to_remove`: added property `aaz_type=AAZGenericUpdateRemoveArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `properties_to_set`: added property `aaz_type=AAZGenericUpdateSetArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config update
> > cmd `datadog sso-config update` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config wait
> > cmd `datadog sso-config wait` update parameter `configuration_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config wait
> > cmd `datadog sso-config wait` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config wait
> > cmd `datadog sso-config wait` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config wait
> > cmd `datadog sso-config wait` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog sso-config wait
> > cmd `datadog sso-config wait` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog sso-config wait
> > cmd `datadog sso-config wait` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` added parameter `log_rules`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` added parameter `metric_rules`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule create
> > cmd `datadog tag-rule create` update parameter `rule_set_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog tag-rule list
> > cmd `datadog tag-rule list` added parameter `pagination_limit`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog tag-rule list
> > cmd `datadog tag-rule list` added parameter `pagination_token`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule list
> > cmd `datadog tag-rule list` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog tag-rule list
> > cmd `datadog tag-rule list` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule list
> > cmd `datadog tag-rule list` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule list
> > cmd `datadog tag-rule list` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog tag-rule list
> > cmd `datadog tag-rule list` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule show
> > cmd `datadog tag-rule show` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog tag-rule show
> > cmd `datadog tag-rule show` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule show
> > cmd `datadog tag-rule show` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule show
> > cmd `datadog tag-rule show` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog tag-rule show
> > cmd `datadog tag-rule show` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule show
> > cmd `datadog tag-rule show` update parameter `rule_set_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` added parameter `log_rules`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` added parameter `metric_rules`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` update parameter `monitor_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` update parameter `monitor_name`: updated property `options` from `['--monitor-name']` to `['--monitor-name', '--name', '-n']`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` update parameter `properties_to_add`: added property `aaz_type=AAZGenericUpdateAddArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` update parameter `properties_to_remove`: added property `aaz_type=AAZGenericUpdateRemoveArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` update parameter `properties_to_set`: added property `aaz_type=AAZGenericUpdateSetArg`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` update parameter `resource_group_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` update parameter `resource_group_name`: added property `type=string`
> > 
> > 
> > 
> > ⚠️ [1010 - ParaPropUpdate](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1010.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` update parameter `resource_group_name`: updated property `name` from `resource_group_name` to `resource_group`
> > 
> > 
> > 
> > ⚠️ [1008 - ParaPropAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1008.md)
> > datadog tag-rule update
> > cmd `datadog tag-rule update` update parameter `rule_set_name`: added property `aaz_type=string`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms create
> > cmd `datadog terms create` added parameter `accepted`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms create
> > cmd `datadog terms create` added parameter `license_text_link`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms create
> > cmd `datadog terms create` added parameter `plan`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms create
> > cmd `datadog terms create` added parameter `privacy_policy_link`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms create
> > cmd `datadog terms create` added parameter `product`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms create
> > cmd `datadog terms create` added parameter `publisher`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms create
> > cmd `datadog terms create` added parameter `retrieve_datetime`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms create
> > cmd `datadog terms create` added parameter `signature`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms list
> > cmd `datadog terms list` added parameter `pagination_limit`
> > 
> > 
> > 
> > ⚠️ [1006 - ParaAdd](https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_rules/1006.md)
> > datadog terms list
> > cmd `datadog terms list` added parameter `pagination_token`

All the parameters using underscore ("_") have been updated with new commands as new codegen tool have the regex condition  [a-z0-9]+(-[a-z0-9]+)*

properties parameter is removed and nested params are added directly.

required = true, there is no change in swagger. These values are added by previous swagger version. Old codegen tool might not have generated this.

cmd datadog terms update removed --> there is no change in swagger. These values are added by previous swagger version. Old codegen tool might have generated but is not being regenerated now. Also it is in experimental phase. Will be updated in future version release.
